cobigen-templates/templates-devon4j/src/main/templates/crud_java_server_app/templates.xml

```diff
@@ -39,6 +40,7 @@
     </increment>
     <increment name="tos" description="TO's">
       <templateRef ref="AbstractSearchCriteriaTo.java"/>
+      <templateRef ref="AbstractComposedKeyEto.java"/>
```

Comment:
As far as I can see, you are forcing the user to always use composited keys when generating Transfer Objects or Entities infrastructure. That should not be the case.

Please create a new increment like `TO's (Composite Key)` and remove here the `AbstractComposedKeyEto.java`. You should follow the same approach as `CTOs`, that they are optional and the user can only generate ETOs.

Reply:
Comment incorporated. Re-review needed.